README.md

```diff
@@ -14,36 +14,53 @@
 
 # Installation Guide
 
-Setup Python environment:
-```
-git clone https://github.com/probcomp/bayes3d.git
-cd bayes3d
+Set up a fresh Python environment:
+
+```bash
 conda create -n bayes3d python=3.9
 conda activate bayes3d
-pip install -r requirements.txt
-pip install -e .
 ```
 
-Install GenJAX (optional):
-```
-pip install git+https://github.com/probcomp/genjax.git@v0.1.0
+Install compatible versions JAX and Torch:
+
+```bash
+pip install --upgrade torch==2.2.0 torchvision==0.17.0+cu118 --index-url https://download.pytorch.org/whl/cu118
+pip install --upgrade jax[cuda11_local]==0.4.20 -f https://storage.googleapis.com/jax-releases/jax_cuda_releases.html
 ```
 
-Install JAX and Torch:
+Bayes3D is built on top of GenJAX, which is currently hosted in a private Python
+package repository. To configure your machine to access GenJAX:
+
+- [File an issue](https://github.com/probcomp/bayes3d/issues/new) asking @sritchie to give you access.
+- [Install the Google Cloud command line tools](https://cloud.google.com/sdk/docs/install).
+- Follow the instructions on the [installation page](https://cloud.google.com/sdk/docs/install)
+- run `gcloud init` as described [in this
+  guide](https://cloud.google.com/sdk/docs/initializing) and configure the tool
+  with the `probcomp-caliban` project ID.
+
+Then run the following command to configure `pip` to use these new gcloud
+commands:
+
+```bash
+pip install keyring keyrings.google-artifactregistry-auth
 ```
-pip install --upgrade "jax[cuda11_pip]" -f https://storage.googleapis.com/jax-releases/jax_cuda_releases.html
-pip install torch torchvision --upgrade --index-url https://download.pytorch.org/whl/cu118
+
+Finally, install Bayes3D:
+
+```bash
+pip install --extra-index-url https://us-west1-python.pkg.dev/probcomp-caliban/probcomp/simple/ \
+    git+https://github.com/probcomp/bayes3d.git@sritchie/inline_deps#egg=bayes3d
 ```
 
 Download model and data assets:
-```
-bash download.sh
-```
 
+```bash
+wget -q -O - https://raw.githubusercontent.com/probcomp/bayes3d/main/download.sh | bash
+```
 
 ## Test
-Run `python demo.py` to test installation setup.
 
+Run `python demo.py` to test installation setup.
```

Comment:
This is still not great, since this requires that a user be inside this repo. @horizon-blue , wdyt of us moving this demo namespace into bayes3d, so that someone can use that namespace to test their install?

Reply:
Sounds reasonable to me. Alternatively, we can also provide a simplified version of the demo.py script in README so people can copy-paste it without cloning the repo, e.g. 

```bash
python -c "import bayes3d as b; b.setup_renderer(b.Intrinsics(height=100, width=100, fx=50.0, fy=50.0, cx=50.0, cy=50.0, near=0.001, far=6.0))"
```

Though admittedly,  this test is not as comprehensive as the full demo script, so moving `demo.py` under bayes3d namespace also sounds good to me (e.g. to `bayes3d/demo.py`), so that we can test installation with `python -m bayes3d.demo`